tests/core/enable-disable-units-gpio/task.yaml

```diff
@@ -57,13 +57,10 @@ execute: |
         exit 0
     fi
 
-    # shellcheck source=tests/lib/journalctl.sh
-    . "$TESTSLIB/journalctl.sh"
-
     echo "Then the snap service units concerning the gpio device must be run before and after a reboot"
     expected="Unit snap.core.interface.gpio-100.service has finished starting up"
-    for i in $(seq 120); do
-        get_journalctl_log -xe --no-pager | MATCH "$expected"
+    for _ in $(seq 120); do
```

Comment:
doesn't shellcheck complain about unused variables in this instance? 

also not for this test, but would be nice to adapt all such instances of `for ... in $(seq XYZ); do ...; sleep ...; done` to use `retry` instead

Reply:
I updated this because I found it weird, but shell check should complain for sure.
Also I'll add to my TODO to update the places where retry could be used.

Reply:
yeah this isn't a big deal, either way you want to handle it is fine